app/assets/javascripts/discourse/models/user.js

```diff
@@ -45,7 +45,8 @@ const User = RestModel.extend({
 
   @discourseComputed("can_be_deleted", "post_count")
   canBeDeleted(canBeDeleted, postCount) {
-    return canBeDeleted && postCount <= 5;
+    const maxPostCount = Discourse.SiteSettings.delete_all_posts_max;
```

Comment:
ditto here

Reply:
`this.siteSettings` is undefined here 🤔

Reply:
Ah yes it's a model. Can't avoid this here for now unless you set it when the model is built. Fine to ignore this one.